underfs/s3/src/main/java/tachyon/underfs/s3/S3UnderFileSystem.java

```diff
@@ -530,6 +531,9 @@ private String stripPrefixIfPresent(String key) {
     if (key.startsWith(mBucketPrefix)) {
       return key.substring(mBucketPrefix.length());
     }
+    if (key.startsWith(PATH_SEPARATOR)) {
+      return key.substring(PATH_SEPARATOR.length());
+    }
     LOG.warn("Attempted to strip key with invalid prefix: " + key);
```

Comment:
Is this log message really true? If the key is unchanged, it will always print this warning?